apps/vc-api/src/vc-api/vc-api.controller.ts

```diff
@@ -248,12 +251,49 @@ export class VcApiController {
     description: 'Verifiable Presentation successfully submitted. Further review in progress.'
   })
   @ApiNotFoundResponse({ type: NotFoundErrorResponseDto })
-  async continueExchange(
+  async continueExchangePut(
+    @Param('exchangeId') exchangeId: string,
+    @Param('transactionId') transactionId: string,
+    @Body() presentation: VerifiablePresentationDto,
+    @Res() res: Response
+  ): Promise<ExchangeResponseDto | Response> {
+    this.logger.warn(`PUT method to be deprecated for exchange continuation`);
+
+    return this.continueExchange({ exchangeId, transactionId, presentation, res });
+  }
+
+  @Post('/exchanges/:exchangeId/:transactionId')
+  @ApiOperation({
+    description:
+      'Receives information related to an existing exchange.\n' +
+      'https://w3c-ccg.github.io/vc-api/#continue-exchange'
+  })
+  @ApiBody({ type: VerifiablePresentationDto })
+  @ApiOkResponse({
+    description: 'Verifiable Presentation successfully submitted and verified',
+    type: ExchangeResponseDto
+  })
+  @ApiAcceptedResponse({
+    description: 'Verifiable Presentation successfully submitted. Further review in progress.'
+  })
+  @ApiNotFoundResponse({ type: NotFoundErrorResponseDto })
+  async continueExchangePost(
     @Param('exchangeId') exchangeId: string,
     @Param('transactionId') transactionId: string,
     @Body() presentation: VerifiablePresentationDto,
     @Res() res: Response
   ): Promise<ExchangeResponseDto | Response> {
+    return this.continueExchange({ exchangeId, transactionId, presentation, res });
+  }
+
+  async continueExchange(options: {
```

Comment:
I personally think it is nice to mark methods as private to reduce the test surface and improve readability.
```suggestion
  private async continueExchange(options: {
```